python/cucim/src/cucim/skimage/feature/tests/test_blob.py

```diff
@@ -0,0 +1,560 @@
+import math
+
+import cupy as cp
+import pytest
+from cupy.testing import assert_almost_equal
```

Comment:
```suggestion
```
`assert_almost_equal` is not available in CuPy (could import from NumPy and copy scalar back to the host first, but we don't need this function at all except in the test cases I suggested deleting below)